src/engine/Events/GameEvent.ts

```diff
@@ -1,85 +1,13 @@
-import { Scene } from './Scene';
-import { Vector } from './Algebra';
-import { Actor } from './Actor';
-import { Trigger } from './Trigger';
-import { FrameStats } from './Debug';
-import { Engine } from './Engine';
-import { TileMap } from './TileMap';
-import { Side } from './Collision/Side';
-import * as Input from './Input/Index';
-import { Pair, BaseCamera } from './Index';
-
-/* istanbul ignore next */
-/* compiler only: these are internal to lib */
-export type kill = 'kill';
-export type prekill = 'prekill';
-export type postkill = 'postkill';
-
-export type predraw = 'predraw';
-export type postdraw = 'postdraw';
-
-export type predebugdraw = 'predebugdraw';
-export type postdebugdraw = 'postdebugdraw';
-
-export type preupdate = 'preupdate';
-export type postupdate = 'postupdate';
-
-export type preframe = 'preframe';
-export type postframe = 'postframe';
-
-export type precollision = 'precollision';
-export type collisionstart = 'collisionstart';
-export type collisionend = 'collisionend';
-export type postcollision = 'postcollision';
-
-export type initialize = 'initialize';
-export type activate = 'activate';
-export type deactivate = 'deactivate';
-
-export type exitviewport = 'exitviewport';
-export type enterviewport = 'enterviewport';
-
-export type exittrigger = 'exit';
-export type entertrigger = 'enter';
-
-export type connect = 'connect';
-export type disconnect = 'disconnect';
-export type button = 'button';
-export type axis = 'axis';
-
-export type subscribe = 'subscribe';
-export type unsubscribe = 'unsubscribe';
-
-export type visible = 'visible';
-export type hidden = 'hidden';
-export type start = 'start';
-export type stop = 'stop';
-
-export type pointerup = 'pointerup';
-export type pointerdown = 'pointerdown';
-export type pointermove = 'pointermove';
-export type pointerenter = 'pointerenter';
-export type pointerleave = 'pointerleave';
-export type pointercancel = 'pointercancel';
-export type pointerwheel = 'pointerwheel';
-
-export type up = 'up';
-export type down = 'down';
-export type move = 'move';
-export type enter = 'enter';
-export type leave = 'leave';
-export type cancel = 'cancel';
-export type wheel = 'wheel';
-
-export type press = 'press';
-export type release = 'release';
-export type hold = 'hold';
-
-export type pointerdragstart = 'pointerdragstart';
-export type pointerdragend = 'pointerdragend';
-export type pointerdragenter = 'pointerdragenter';
-export type pointerdragleave = 'pointerdragleave';
-export type pointerdragmove = 'pointerdragmove';
+import { Scene } from '../Scene';
```

Comment:
Let's not refactor `Events.ts` at this time. It causes a lot of changes in files unrelated to audio.